cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -23,7 +23,7 @@
 "containers":[
     {
     "name": "cloud-controller-manager",
-    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v29.0.0",
+    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v0.30.0-alpha",
```

Comment:
Is this a temporary tag?

- v29.0.0
- v0.30.0-alpha

The tags look quite different

Reply:
this is for testing, sorry, I just sent it yesterday quickly, and I found out the fix is wrong